pkg/namesgenerator/names-generator.go

```diff
@@ -730,6 +730,9 @@ var (
 
 		// Françoise Barré-Sinoussi - French virologist and Nobel Prize Laureate in Physiology or Medicine; her work was fundamental in identifying HIV as the cause of AIDS. https://en.wikipedia.org/wiki/Fran%C3%A7oise_Barr%C3%A9-Sinoussi
 		"sinoussi",
+		
+		// Johnny Sins - He is a respected doctor, an engineer, karate master, pizza delivery guy, plumber, he had his own company, was in the army and fought for his country, he was a professor, a detective, a cop, a wrestling champion and an astronaut. So he is famous because he is the most talented man alive on earth.
+		"sins",
```

Comment:
Unless there's another person named "Johnny Sins" (in which case you should include a link in the comment), I don't think Porn star really fits the bill https://en.wikipedia.org/wiki/Johnny_Sins

https://github.com/moby/moby/blob/e952346c9976d365b74588daf2cb886fdbd1b8ce/pkg/namesgenerator/names-generator.go#L120-L121